### PR TITLE
Week 1 assignments

### PR DESCRIPTION
**Pull request by ayinoop**
Hi Sai,

Pull request for week 1 assignment. I use libreoffice so the odd file formats are from libredraw and librewrite.

Thanks,
Naveen Pooniya

### CODE REVIEW

**Review by sairamin — CHANGES_REQUESTED** [review]
Points to remember:

- Separate PR's for each assignment
- Follow camelCase for file/folder naming

File: Week-1/cattyboard/cattyboard.htm [hunk 1]
```diff
@@ -0,0 +1,187 @@
+<html>
+<style>
```
Comment (sairamin) [inline Week-1/cattyboard/cattyboard.htm:2]: Rename to .html

Read about Doctype.

File: Week-1/cattyboard/cattyboard.htm [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  * {
+    box-sizing: border-box;
```
Comment (sairamin) [inline Week-1/cattyboard/cattyboard.htm:4]: Read about different ways of adding css.

File: Week-1/cattyboard/cattyboard.htm [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    float: left;
+    width: 3em;
+    height:85%;
+    background: #a9a9a9;
+    padding: 10px 10px;
+  }
+  li button {
+
+    border: None;
+    outline: None;
```
Comment (sairamin) [inline Week-1/cattyboard/cattyboard.htm:29]: None vs none ?
What is the difference ?

File: Week-1/cattyboard/cattyboard.htm [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+    background: none;
+
+  }
+  p button {
+    padding-left:2em;
```
Comment (sairamin) [inline Week-1/cattyboard/cattyboard.htm:34]: Why did you use em and why not px or rem ?

File: Week-1/cattyboard/cattyboard.htm [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+    color: white;
+  }
+
+  nav {
+    position: absolute;
+    float: left;
+    width: 3em;
+    height:85%;
+    background: #a9a9a9;
+    padding: 10px 10px;
```
Comment (sairamin) [inline Week-1/cattyboard/cattyboard.htm:24]: padding: 10px

File: Week-1/cattyboard/cattyboard.htm [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+        }
+  section:after {
+    content: "";
+    display: table;
+    clear: both;
+  }
+
+  footer {
+    position: absolute;
+    bottom: 0px;
```
Comment (sairamin) [inline Week-1/cattyboard/cattyboard.htm:77]: consistency

File: Week-1/cattyboard/cattyboard.htm [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+    height:15%;
+    width: 100%;
```
Comment (sairamin) [inline Week-1/cattyboard/cattyboard.htm:79]: 100% vs 100vh ?

File: Week-1/cattyboard/cattyboard.htm [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+    vertical-align: bottom;
+    padding: 20px;
+    text-align: center;
+    color: white;
+  }
+  li {
+
+    padding-top:4em;
+  }
+  .chartimg {
```
Comment (sairamin) [inline Week-1/cattyboard/cattyboard.htm:90]: spelling mistake

File: Week-1/cattyboard/cattyboard.htm [hunk 9]
```diff
@@ -0,0 +1,187 @@
...
+  .charttext {
+    float: left;
+    padding: 120px;
+    width: 80%;
+    height: 80px;
+    display: flex;
+    display: table-cell;
+    vertical-align: top;
+  }
+</style>
```
Comment (sairamin) [inline Week-1/cattyboard/cattyboard.htm:116]: Why is style outside of head ?

File: Week-1/cattyboard/cattyboard.htm [hunk 10]
```diff
@@ -0,0 +1,187 @@
...
+<head>
+<title>Font Awesome Icons</title>
+<meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (sairamin) [inline Week-1/cattyboard/cattyboard.htm:119]: What happens if i remove this ?

File: Week-1/cattyboard/cattyboard.htm [hunk 11]
```diff
@@ -0,0 +1,187 @@
...
+<link rel="stylesheet" href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
+</head>
+<body>
+  <h1></h1>
```
Comment (sairamin) [inline Week-1/cattyboard/cattyboard.htm:123]: Empty tag ?

File: Week-1/cattyboard/cattyboard.htm [hunk 12]
```diff
@@ -0,0 +1,187 @@
...
+          <td>Work from Home</td>
+          <td>Cat Harmony feat Colla</td>
+          <td>3:34</td>
+          <td><Button>CATTY CLOUD SYNC</Button></td>
+        </tr>
+      </table>
+    </article>
+  </section>
+  <footer>
+  <div class="fchartimg"><img src="https://vif1g.csb.app/src/assets/cattyboard.jpg" height=75em width=75em align: "left"></div>
```
Comment (sairamin) [inline Week-1/cattyboard/cattyboard.htm:180]: why inline css ?

Comment (sairamin) [reply Week-1/cattyboard/cattyboard.htm:180]: Why can't we directly add class to the img tag ?

Comment (sairamin) [reply Week-1/cattyboard/cattyboard.htm:180]: Start using a linter.

Comment (sairamin) [reply Week-1/cattyboard/cattyboard.htm:180]: Always use a linter.

Comment (ayinoop) [reply Week-1/cattyboard/cattyboard.htm:180]: Inline Css to resize the footer image.
We can add but i wanted to keep image element in separate div for future linkage.
configured linter on atom.

File: Week-1/cattyboard/cattyboard.htm (hunk 11 above)
Comment (ayinoop) [reply Week-1/cattyboard/cattyboard.htm:123]: deleted

File: Week-1/cattyboard/cattyboard.htm (hunk 10 above)
Comment (ayinoop) [reply Week-1/cattyboard/cattyboard.htm:119]: On laptop browser there wasn't any visible difference, upon further reading on meta tag, got to know about its function and use, which is the set viewport content width as device width and initial zoom level as 1.0.

File: Week-1/cattyboard/cattyboard.htm (hunk 9 above)
Comment (ayinoop) [reply Week-1/cattyboard/cattyboard.htm:116]: It still seems to work, however now moved inside head tags.

File: Week-1/cattyboard/cattyboard.htm (hunk 7 above)
Comment (ayinoop) [reply Week-1/cattyboard/cattyboard.htm:79]: vh and vw applied, first is 100% of parent element width, while later is 100% of viewport width.

File: Week-1/cattyboard/cattyboard.htm (hunk 6 above)
Comment (ayinoop) [reply Week-1/cattyboard/cattyboard.htm:77]: maintained 0px;

File: Week-1/cattyboard/cattyboard.htm (hunk 5 above)
Comment (ayinoop) [reply Week-1/cattyboard/cattyboard.htm:24]: Updated

File: Week-1/cattyboard/cattyboard.htm (hunk 4 above)
Comment (ayinoop) [reply Week-1/cattyboard/cattyboard.htm:34]: Associated with P tag font size.

File: Week-1/cattyboard/cattyboard.htm (hunk 3 above)
Comment (ayinoop) [reply Week-1/cattyboard/cattyboard.htm:29]: Although HTML is not case sensitive, some browser may restrict the case sensitivity. It is better to maintain consistency. Preference to lower case than sentence case.

File: Week-1/cattyboard/cattyboard.htm (hunk 2 above)
Comment (ayinoop) [reply Week-1/cattyboard/cattyboard.htm:4]: read, Inline for single element, internal for the page its defined and external can be used across pages.

File: Week-1/cattyboard/cattyboard.htm (hunk 1 above)
Comment (ayinoop) [reply Week-1/cattyboard/cattyboard.htm:2]: Done.
Added Doctype declaration at top.

**Pull request closed without merging**